repose-aggregator/core/repose-core/src/main/java/org/openrepose/powerfilter/PowerFilterRouter.java

```diff
@@ -22,11 +22,11 @@
 import org.openrepose.commons.utils.servlet.http.HttpServletRequestWrapper;
 
 import javax.servlet.ServletException;
-import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import java.io.IOException;
 import java.net.URISyntaxException;
 
+@FunctionalInterface
```

Comment:
Questionable, i don't think that's the intent here, it's certainly not the usage. An argument could be made for it to be a lambda like expression though